wh10/js/main.js

```diff
@@ -0,0 +1,23 @@
+// Створити масив, довжину та елементи якого задає користувач.
+let arr = [];
+let el;
+do{
+    el = prompt('Enter nuber','');
```

Comment:
Було б добре повидомити користувачу, як завершити ввод елементів
```suggestion
    el = prompt('Enter array element, press "Esc" when finished','');
```